CRM/Utils/Check/Component/Env.php

```diff
@@ -1120,21 +1120,60 @@ public function checkMysqlVersion() {
     return $messages;
   }
 
-  public function checkPHPIntlExists() {
+  public function checkPHPIntlExists(): array {
     $messages = [];
     if (!extension_loaded('intl')) {
       $messages[] = new CRM_Utils_Check_Message(
         __FUNCTION__,
         ts('This system currently does not have the PHP-Intl extension enabled.  Please contact your system administrator about getting the extension enabled.'),
         ts('Missing PHP Extension: INTL'),
-        \Psr\Log\LogLevel::WARNING,
+        LogLevel::WARNING,
         'fa-server'
       );
     }
     return $messages;
   }
 
-  public function checkAngularModuleSettings() {
+  /**
+   * Let people know if they could improve performance by defining the exclude directory.
+   *
+   * @return array
+   */
+  public function checkExcludeDirectories(): array {
+    $messages = [];
+    if (!defined('CIVICRM_EXCLUDE_DIRS_PATTERN')) {
```

Comment:
This might create some queries from windows people. It's not that it's not possible to use it, but there was some difficulty getting the right regex, so this is always disabled in civicrm.settings.php for windows, so this will come up on every windows site. But at the same time if this prompts some windows people to figure out a good regex that would be a good thing, so I'm 50/50 on whether this should have the same guard as in civicrm.settings.php. 🤔